manifests/fpm/config.pp

```diff
@@ -93,6 +93,7 @@
   String[1] $root_group                                                 = $php::params::root_group,
   String $syslog_facility                                               = 'daemon',
   String $syslog_ident                                                  = 'php-fpm',
+  Boolean $manage_run_dir                                               = true
```

Comment:
Please add documentation for this new parameter in the comment block above. That way it will be added to REFERENCE.md on the next release.